scripts/ticketmaster-API.py

```diff
@@ -17,63 +17,96 @@
 
 URL = "https://app.ticketmaster.com/discovery/v2/events.json"
 
-us_states = ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE",
-             "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS",
-             "KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS",
-             "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY",
-             "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC",
-             "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY"]
+us_states = [#"AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE",
+             #"FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS",
+             #"KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS",
+             #"MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY",
+             #"NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC",
+             #"SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI",
+             "NY"]
 
 apikey = "sDeUMZRAoiaKBkELqJqSCS2phLMxJj4A"
+#apikey2 = "s10q0X1HOmg4rgAtNS5wuhgekEcFaGYC"
```

Comment:
Let's move this value to the config file (in the same way we store the mysql variables). You could get it by doing:
```python
apikey = config['ticketmaster']['apikey']
```
And in `config.ini` you would add:
```
[mysql]
...
[ticketmaster]
apikey = s10q0X1HOmg4rgAtNS5wuhgekEcFaGYC
```

Reply:
Put both of them in config.ini.